jenkins/jobs/macros-common.yaml

```diff
@@ -60,6 +60,16 @@
           #echo "Detailed logs: http://logs.openstack.xenproject.org/$LOG_PATH/"
           echo "Detailed logs: http://dd6b71949550285df7dc-dda4e480e005aaa13ec303551d2d8155.r49.cf1.rackcdn.com/$LOG_PATH/"
 
+- builder:
+    name: copy-dom0-logs
+    builders:
+      - shell: |
+          #!/bin/sh
+          # make dir on jekins server to save Dom0 logs
+          sudo -u domzero mkdir -p ~domzero/logs
+          sudo -u domzero scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@{dom0_ip}:/var/log/\{messages*,user.log*,SMlog*,xensource.log\} /home/domzero/logs/
+          sudo -u domzero chmod -R 644 ~domzero/logs
```

Comment:
I guess this will fail. You shouldn't remove the executable bit for a directory, otherwise it's not able to enter this directory.